src/app/hooks/useTips.test.ts

```diff
@@ -32,14 +32,14 @@ jest.mock("~/app/context/AccountContext", () => ({
 
 describe("useTips", () => {
   test("should not have top up wallet tip when using a non-alby account", async () => {
-    tmpAccount = { id: "1", name: "LND account", alias: "" };
+    tmpAccount = { id: "1", name: "LND account", connector: "" };
     const { tips } = useTips();
     expect(tips.length).toBe(1); // mnemonic
     const hasTopUpWallet = tips.some((tip) => tip === TIPS.TOP_UP_WALLET);
     expect(hasTopUpWallet).toBe(false);
   });
   test("should have top up wallet tip when having alby account", async () => {
-    tmpAccount = { id: "2", name: "Alby", alias: "🐝 getalby.com" };
```

Comment:
should we leave in tests for old Alby lndhub account? (where `isAlbyLNDHubAccount` returns true)

Reply:
added one more test for legacy lndhub alby accounts. where alias being checked and card being showed